attributes/default.rb

```diff
@@ -144,7 +136,7 @@
 default['percona']['server']['log_bin_basename'] = 'master-bin' # 5.6~> default: datadir + '/' + hostname + '-bin'
 default['percona']['server']['relay_log'] = 'slave-relay-bin'
 default['percona']['server']['log_slave_updates'] = false
-default['percona']['server']['log_warnings'] = true
+default['percona']['server']['log_warnings'] = !(node['percona']['version'].to_f >= 8.0)
```

Comment:
I feel like from a legibility this should be a `true unless` instead of this, both being valid solutions, this just takes a little longer to comprehend

Reply:
I think cookstyle fixed it for me to use this. Perhaps this might be more readable?
```suggestion
default['percona']['server']['log_warnings'] = node['percona']['version'].to_f < 8.0
```